### DISCUSSION TITLE
Theme Flat & Shinne

### DISCUSSION THREAD

**Discussion by jujuk212** [notrinos/NotrinosERP#5] [General]
Hai.

Thank you for sharing notrinosERP with us.
In the demo I tried the flat and shinne themes, but after I downloaded version 0.1 I didn't find the flat and shinne themes, are these themes not suitable for sharing?

Thank & Best Regards

**Comment by notrinos** [comment]
Hi @jujuk212 
Did you find those themes from a topic that is not related to NotrinosERP ?
There has been no such themes for NotrinosERP to now.
